packages/server-wallet/src/protocols/challenge-submitter.ts

```diff
@@ -38,10 +38,10 @@ export class ChallengeSubmitter {
         return;
       }
 
-      if (
-        channel.challengeStatus &&
-        channel.challengeStatus.toResult().status !== 'No Challenge Detected'
-      ) {
+      const status = channel.adjudicatorStatus
+        ? channel.adjudicatorStatus.toResult().status
+        : 'Nothing';
+      if (status !== 'Nothing') {
```

Comment:
I think you're right. I think this is the correct check to do since if there is a `finalized` channel (from either a challenge or conclude) we don't want to submit a challenge. So I think the warning needs to be updated.

Reply:
The warning has been updated in d33e8bd

Reply:
Cool. I guess there are scenarios where you might want to [submit another challenge](https://docs.statechannels.org/docs/protocol-tutorial/clear-a-challenge#call-challenge-again) when there is one ongoing -- but that is probably out of scope for this PR. Just wanted to mention, for completeness.